src/main/asciidoc/java/http.adoc

```diff
@@ -1251,24 +1251,24 @@ client.websocket("/some-uri", websocket -> {
 
 ==== Writing messages to WebSockets
 
-If you wish to write a single binary WebSocket message containing a single WebSocket frame to the WebSocket (a
-common case) the simplest way to do this is to use `link:../../apidocs/io/vertx/core/http/WebSocket.html#writeMessage-io.vertx.core.buffer.Buffer-[writeMessage]`:
+If you wish to write a single binary WebSocket message to the WebSocket you can do this with
+`link:../../apidocs/io/vertx/core/http/WebSocket.html#writeBinaryMessage-io.vertx.core.buffer.Buffer-[writeBinaryMessage]`:
```

Comment:
Sorry I don't follow which part of this comment is misleading:
"If you wish to write a single binary WebSocket message to the WebSocket you can do this with"
It seems correct to me.
Binary message and binary frame are different things.